src/Microsoft.Azure.WebJobs.Host/Indexers/MethodInfoExtensions.cs

```diff
@@ -16,7 +16,9 @@ public static string GetFullName(this MethodInfo methodInfo)
                 throw new ArgumentNullException("methodInfo");
             }
 
-            return String.Format(CultureInfo.InvariantCulture, "{0}.{1}", methodInfo.DeclaringType.FullName, methodInfo.Name);
+            return TryGetCustomName(methodInfo, out string customName)
```

Comment:
I thought I overheard Fabio saying appVeyor supported C#7. I probably miss understood, I'll fix it.